Pages/Incoming.cshtml

```diff
@@ -42,7 +42,7 @@
                     @statusIcon We are @(incoming.CommitDistance == null ? "(unknown)" : incoming.CommitDistance == 0 ? "0" : $"{incoming.CommitDistance}") commit(s) behind
                 </h5>
                 <p class="card-text">
-                    Oldest unconsumed commit was @incoming.CommitAge.Humanize()
+                    Oldest unconsumed commit was @(incoming.CommitAge == null ? "(unknown)" : incoming.CommitAge.Humanize())

```

Comment:
Do you need a ".Value" here, or does `Humanize()` handle it?

Reply:
`Humanize` has an overload for nullable `DateTimeOffset` :+1: